### PR TITLE
[TEVA-1942] show/hide radius select

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-1942

### CODE REVIEW

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2850.london.cloudapps.digital>

File: app/components/jobseekers/search_results/job_alerts_link_component.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class Jobseekers::SearchResults::JobAlertsLinkComponent < ViewComponent::Base
+  attr_accessor :coordinates_present
```
Comment (david-mears-dfe) [inline app/components/jobseekers/search_results/job_alerts_link_component.rb:2]: I don't think this attribute is ever set or read.

**Comment by david-mears-dfe** [comment]
The behaviour looks good, but I have commented on some code that appears not to do anything.

File: app/components/jobseekers/search_results/job_alerts_link_component.rb (hunk 1 above)
Comment (alexbowen) [reply app/components/jobseekers/search_results/job_alerts_link_component.rb:2]: good spot, i added this when trying something out

File: app/components/jobseekers/search_results/job_alerts_link_component/job_alerts_link_component.html.slim [hunk 2]
```diff
@@ -2,4 +2,4 @@
   .job-alert-cta__fade
   .job-alert-cta__content class="govuk-heading-s govuk-!-font-weight-regular govuk-!-margin-0" id="job-alert-cta"
     .icon.icon--left.icon--alert-white
-      = t("subscriptions.link.help_text_html", link: link_to(t("subscriptions.link.text"), new_subscription_path(search_criteria: @vacancies_search.active_criteria, origin: @origin), id: "job-alert-link-sticky-gtm"))
+      = t("subscriptions.link.help_text_html", link: link_to(t("subscriptions.link.text"), new_subscription_path(search_criteria: @vacancies_search.active_criteria, origin: @origin, coordinates_present: @vacancies_search.point_coordinates.present?), id: "job-alert-link-sticky-gtm"))
```
Comment (cpjmcquillan) [inline app/components/jobseekers/search_results/job_alerts_link_component/job_alerts_link_component.html.slim:5]: I think we need a `&` before `.point_coordinates`. Try searching for a location polygon 🤯

Comment (alexbowen) [reply app/components/jobseekers/search_results/job_alerts_link_component/job_alerts_link_component.html.slim:5]: hmm im not seeing a problem? speak in the morning

Comment (cpjmcquillan) [reply app/components/jobseekers/search_results/job_alerts_link_component/job_alerts_link_component.html.slim:5]: @alexbowen https://teaching-vacancies-review-pr-2850.london.cloudapps.digital/jobs?keyword=&location=london&job_roles%5B%5D=&phases%5B%5D=&working_patterns%5B%5D=&commit=Search

**Review by cpjmcquillan — APPROVED** [review]

**Pull request merged**